packages/datadog_grpc_interceptor/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   datadog_flutter_plugin: ^2.7.0
-  grpc: ^3.0.2
+  grpc: ^4.0.0
```

Comment:
Looks like there are no changes in v4 that are breaking for this interface. Do you think we could do ">=3.0.2<5.0.0"?

Reply:
Yeah, that's fine. I tested it on the project I'm working on with a dependency override and both gRPC versions were compatible.